scripts/delete-istio.sh

```diff
@@ -2,6 +2,7 @@
 
 # Initialization
 SCRIPT_DIR=$(dirname "$0")
+# shellcheck source=scripts/init-env.sh
```

Comment:
If you add `SHELLCHECK_OPTS: --external-sources` you don't need this line. I agree to have the exception.

Reply:
I tried locally and it's required due to: `ShellCheck is unable to follow dynamic paths, such as source "$somedir/file"`. See [SC1091](https://www.shellcheck.net/wiki/SC1091).